src/main/resources/lib/convert/convert.ts

```diff
@@ -210,10 +210,11 @@ function dataqueryToDataSource(dataquery: Content<Dataquery>): DataSource['dataS
 }
 
 export interface Socket {
```

Comment:
Maybe move generic socket interfaces to "src\main\resources\lib\types"?

Reply:
Moved to "src/main/resources/lib/types/socket.ts"?